packages/tfjs-node-helpers/src/testing/metrics/brier-loss.ts

```diff
@@ -0,0 +1,21 @@
+import { metrics } from '@tensorflow/tfjs-node';
+import { Metric } from '../metric';
+import { MetricCalculator, MetricCalculatorData } from '../metric-calculator';
+
+export class BrierLossMetricCalculator extends MetricCalculator {
+  public calculate({ trueValues, probabilities }: MetricCalculatorData): Metric {
+    const trueY = trueValues.as1D();
+    const predY = probabilities.as1D();
+    const valueTensor = metrics.MSE(trueY, predY);
+    const value = valueTensor.dataSync()[0];
+
+    trueY.dispose();
+    predY.dispose();
```

Comment:
@VladimirGlinskii It looks like we can dispose `trueY` and `predY` before calling `valueTensor.dataSync()`.